### PR TITLE
Ddf 2481 2.9.x Updated Installation & quickstart instructions

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Copy edits to quickstart and installation instructions.

#### Who is reviewing it?

@mcalcote @clockard @tbatie @garrettfreibott @emanns95 @mweser @spearskw 

#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).

[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR.

@pklinef
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

Verify changes to content.

#### What are the relevant tickets?
[DDF-2481](https://codice.atlassian.net/browse/DDF-2481)

#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/quickstart-contents.adoc [hunk 1]
```diff
@@ -9,35 +15,55 @@ These steps will demonstrate:
...
-** Replace `<JAVA_VERSION>` with the version and build number installed.
+
+.Setting JAVA_HOME variable (Replace `<JAVA_VERSION>` with the version and build number installed.)
+====
+
+. Determine Java Installation Directory (This varies between operating system versions).
++
+.Find Java Path
+----
+which java
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/_contents/quickstart-contents.adoc]: Is this line right?

File: distribution/docs/src/main/resources/_contents/quickstart-contents.adoc [hunk 2]
```diff
@@ -9,35 +15,55 @@ These steps will demonstrate:
...
+----
++
+. Copy path to Java installation. (example: `/usr/java/<JAVA_VERSION`)
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/_contents/quickstart-contents.adoc]: Java version needs a `>` at the end

File: distribution/docs/src/main/resources/_contents/quickstart-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/quickstart-contents.adoc]: yep. that's the command to retrieve your path.

Comment (mcalcote) [inline distribution/docs/src/main/resources/_contents/quickstart-contents.adoc]: 👍

File: distribution/docs/src/main/resources/_contents/quickstart-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/quickstart-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/quickstart-contents.adoc [hunk 3]
```diff
@@ -9,35 +15,55 @@ These steps will demonstrate:
...
 setx PATH "%PATH%;%JAVA_HOME%\bin" /m
 ----
+====
 
 .*NIX
 [WARNING]
 ====
-Unlink `/usr/bin/java` if it is already linked to a previous version of the JRE:
-`unlink /usr/bin/java`
+Unlink `<PATH_TO_JAVA` if it is already linked to a previous version of the JRE:
```
Comment (emanns95) [inline distribution/docs/src/main/resources/_contents/quickstart-contents.adoc]: Close angle-bracket >

Comment (emanns95) [inline distribution/docs/src/main/resources/_contents/quickstart-contents.adoc]: And shouldn't this be JAVA_HOME?

File: distribution/docs/src/main/resources/_contents/quickstart-contents.adoc [hunk 4]
```diff
@@ -9,35 +15,55 @@ These steps will demonstrate:
...
+
+`unlink <PATH_TO_JAVA`
```
Comment (emanns95) [inline distribution/docs/src/main/resources/_contents/quickstart-contents.adoc]: Close angle-bracket >, JAVA_HOME?

File: distribution/docs/src/main/resources/_contents/quickstart-contents.adoc [hunk 5]
```diff
@@ -76,22 +99,42 @@ init 6
...
 . Install ${branding} by unzipping the zip file.
++
+.Windows Zip Utility Warning
+[WARNING]
+====
+The default Windows zip utility (such as double-clicking file) will not work to unzip the distribution, use Java or a third party utility instead.
+
+.Use Java to Unzip in Windows(Replace `<JAVA_VERSION>` with Current Version)
+----
+"C:\Program Files\Java\jdk<JAVA_VERSION>\bin\jar.exe" xf ${branding-lowercase}-${project.version}.zip
```
Comment (emanns95) [inline distribution/docs/src/main/resources/_contents/quickstart-contents.adoc]: Should we use <PATH_TO_JAVA> here as well?

File: distribution/docs/src/main/resources/_contents/quickstart-contents.adoc (hunk 4 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/quickstart-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/configuring-from-the-command-console-contents.adoc [hunk 6]
```diff
@@ -146,9 +147,7 @@ State         Version          Name                          Repository  Descrip
 [uninstalled] [${project.version}         ] ${ddf-branding-lowercase}-source-dummy              ddf-${project.version}
 ----
 
-===== Configuring HTTP Port from the ${command-console}
-
-==== Configuring HTTP to HTTPS Proxy From the ${command-console}
+===== Configuring HTTP to HTTPS Proxy From the ${command-console}
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/configuring-from-the-command-console-contents.adoc]: Should we continue to document the HTTP Proxy? @codice/security

File: distribution/docs/src/main/resources/_contents/installing-contents.adoc [hunk 7]
```diff
@@ -16,29 +19,43 @@ These are the system/environment requirements to configure _prior_ to an install
...
 * http://www.oracle.com/technetwork/java/javase/downloads/index.html[JDK8] must be installed.
 * The `JAVA_HOME` environment variable must be set to the location where the JDK is installed.
-** Replace `<JAVA_VERSION>` with the the version and build number installed.
+
+.Setting JAVA_HOME variable (Replace `<JAVA_VERSION>` with the version and build number installed.)
+====
+
+. Determine Java Installation Directory (This varies between operating system versions).
++
+.Find Java Path
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/installing-contents.adoc]: Should we explicitly state this on `*NIX*, like other commands in this section?

File: distribution/docs/src/main/resources/_contents/installing-contents.adoc [hunk 8]
```diff
@@ -16,29 +19,43 @@ These are the system/environment requirements to configure _prior_ to an install
...
 setx PATH "%PATH%;%JAVA_HOME%\bin" /m
 ----
+====
 
 .*NIX
 [WARNING]
 ====
-Unlink `/usr/bin/java` if it is already linked to a previous version of the JRE:
-`unlink /usr/bin/java`
+Unlink `<PATH_TO_JAVA` if it is already linked to a previous version of the JRE:
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/installing-contents.adoc]: Are we missing `>` on `<PATH_TO_JAVA` on line 56 and 58?

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/installing-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/installing-contents.adoc (hunk 7 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/installing-contents.adoc]: Yes. Added Windows version.

File: distribution/docs/src/main/resources/draft-documentation.adoc [hunk 9]
```diff
@@ -0,0 +1,358 @@
+= DRAFT ${branding-expanded} Documentation
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/draft-documentation.adoc]: Were we planning on keeping this draft for 2.9.4?

Comment (ricklarsen) [inline distribution/docs/src/main/resources/draft-documentation.adoc]: Actually, it shouldn't even be here. Removed.

File: distribution/docs/src/main/resources/_contents/configuring-from-the-command-console-contents.adoc (hunk 6 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/configuring-from-the-command-console-contents.adoc]: removed

**Pull request merged**